xonsh/xoreutils/__init__.py

```diff
@@ -0,0 +1,11 @@
+import importlib
+
+from xonsh.xoreutils.util import all_builtin_commands
+
+cmd_names = {'cat', 'pwd', 'tee', 'tty', 'yes', 'echo', 'which',
+             'enable'}
+
+
+for i in cmd_names:
+    cmd = importlib.__import__('xonsh.xoreutils.%s' % i, globals(), locals(), [i], 0)
```

Comment:
Also I think you should be using [`import_module()`](https://docs.python.org/3/library/importlib.html#importlib.import_module) as per [`__import__()`](https://docs.python.org/3/library/importlib.html#importlib.__import__)'s docstring.